cmd/list/operatorroles/cmd.go

```diff
@@ -188,20 +188,36 @@ func run(cmd *cobra.Command, _ []string) {
 			r.Reporter.Infof(noOperatorRolesPrefixOutput)
 			os.Exit(0)
 		}
-		fmt.Fprintf(writer, "ROLE NAME\tROLE ARN\tCLUSTER ID\tVERSION\tMANAGED\n")
+		hasClusterUsingOperatorRolesPrefix, err := r.OCMClient.HasAClusterUsingOperatorRolesPrefix(args.prefix)
+		if err != nil {
+			r.Reporter.Errorf("There was a problem checking if any clusters"+
+				" are using Operator Roles Prefix '%s' : %v", args.prefix, err)
+			os.Exit(1)
+		}
+
+		fmt.Fprintf(writer, "OPERATOR NAME\tOPERATOR NAMESPACE\tROLE NAME\t"+
+			"ROLE ARN\tCLUSTER ID\tVERSION\tPOLICIES\tMANAGED\tIN USE\n")
```

Comment:
To be consistent with `rosa list account-roles` we can change to:
```suggestion
			"ROLE ARN\tCLUSTER ID\tVERSION\tPOLICIES\tAWS Managed\tIN USE\n")
```